### PR TITLE
updated to 7.10.1

### PR DESCRIPTION
**Pull request by sadensmol**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -26,16 +26,15 @@ Pull requests and issues are welcome.
...
 - [AIR SDK](http://www.adobe.com/devnet/air/air-sdk-download.html)
 
 ##### Building:
-- Add the path to your AIR SDK in `build.config` (under `air.sdk`)
+- Make a `build.config` by copying `build.config.dist` (your local config file)
+- Change the path to the AIR SDK in `build.config` (named `air.sdk`)
 - From the root of the repository, run `ant`.
 
-After build completes, the native extension will be in `bin/AdMob.ane`.
+After build completes, the native extension will be in `bin/UnityAds.ane`.
```
Comment (ekilah) [inline README.md]: AdMob

File: .gitignore [hunk 2]
```diff
@@ -4,6 +4,7 @@ build/
 
 # Other files and folders
 .settings/
+.build.config.dist
```
Comment (ekilah) [inline .gitignore]: the `.dist` should be in the repo, and `build.config` should be ignored. then, the workflow is:
- clone repo 
- duplicate `.dist` file and remove `.dist` extension
- modify (gitignored) new file, this is what the build uses

**Comment by ekilah** [comment]
did the new ANE work well with Heyzap SDK 9 @sadensmol ?

**Comment by sadensmol** [comment]
Fixed according to comments.
New ANE works on ios devices without any critical errors.

I've created a new branch for testing new ANEs (  new admob and google ads anes are included )
https://github.com/Heyzap/flex-example-app/tree/ane_updating

File: build.config [hunk 3]
```diff
@@ -1,5 +1,5 @@
 # Path to AIR SDK
```
Comment (ekilah) [inline build.config:1]: this file should be removed from the repo completely

**Comment by ekilah** [comment]
this works, besides my comment about the build.config

**Comment by ekilah** [comment]
one thing - the ANE you put in the test app branch is 49 MB - is that right? seems pretty large.

**Comment by sadensmol** [comment]
@ekilah added needed changes. Also 49mb is correct. It has .framework with some symbol links which change to normal files after copying.

**Pull request closed without merging**